apiserver/grpc/server.go

```diff
@@ -115,14 +127,17 @@ func (g *GRPCServer) RollbackStream(req *proto.RollbackPistageRequest, stream pr
 		return err
 	}
 
+	ctx, cancel := context.WithTimeout(stream.Context(), g.timeout)
```

Comment:
For stream funcs, no need to set a timeout because it is fully determined by when the caller ends the connection.